File: TypeScript/Dividers/dividers.ts
export default function dividers(number: number): void 
{
    let numCount, numDiv = 0;
    numDiv = number;
    while (numDiv > 0) {
        numCount = number % numDiv;
        if (numCount == 0) {
            console.log(numDiv);
        }
        numDiv--;
    }
    return;
}

File: C#/DividersEnum/DividersEnum/Dividers.cs
namespace DividersEnum
{
    internal class Dividers
    {
        public static IEnumerable<int> DividersFunc(int num)
        {
            int numDiv, numCount = 0;
            numDiv = num;

            while (numDiv > 0)
            {
                numCount = num % numDiv;
                if (numCount == 0)
                {
                    yield return numDiv;
                }
                numDiv--;
            }
        }
    }
}
